packages/ethereum/web3-v4-ethereum/src/utils/parse-request-error/index.ts

```diff
@@ -0,0 +1,14 @@
+import type { ProviderRequestError } from "./domain"
+import { ethereumProviderErrors, ethereumRpcErrors } from "./domain"
+
+export function parseRequestError(error: any): ProviderRequestError | undefined {
+	if (typeof error === "object" && "code" in error) {
```

Comment:
!!! it will be exception in case error: null, make sure you didn't have same mistake somewhere else

typeof error === "object" must be always 

```
typeof error === "object" && error !== null
```

since if you trying to read property in null it's an exception